tests/platform_tests/api/test_chassis_fans.py

```diff
@@ -216,8 +262,17 @@ def test_set_fans_led(self, duthosts, enum_rand_one_per_hwsku_hostname, localhos
             "amber",
             "green",
         ]
+        duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+        fans_skipped = 0
 
         for i in range(self.num_fans):
+            led_controllable = self.get_fan_facts(duthost, i, True, "status_led", "controllable")
+            if not led_controllable:
```

Comment:
for platform that does not support capability query this check will cause their tests be skipped as well?

Reply:
never mind my comments... I see there is a default value involved with the call so it should be fine...